src/App/ComponentsDocumentation/components/Loader/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+import { ComponentPreview, DocContainer } from "@docutils";
+import { loaderShowCase, howToUse } from "./constants";
+
+const Overview = () => (
+    <section id="loader-page-overview">
+        <h2 id="overview">Overview</h2>
+        <ComponentPreview language="html" showCasePanel showCasePanelAdvanced={loaderShowCase} codeFigure removeOuterTag/>
+
+        <section id="loaders-how-to-use">
+            <h3>How to use loader in context</h3>
+            <div className="row">
+                {howToUse.map(({ contents, description }, i) => (
+                    <React.Fragment key={i}>
```

Comment:
even though its necessary when there's a key, I actually kind of like the verbosity of writing out `React.Fragment` instead of the sugar syntaxed `<>` because of the highlighting and readability it gets 🤔

Reply:
It does makes it more readable. You suggest we from now on should write `<React.Fragment>` instead of `<>` ?

Reply:
Hmm, not sure :) 
lets skip it for now